Migrate user guide/examples to mkdocs

qiaojunfeng opened the pull request:
- The original latex code is converted to markdown files
- `examples` renamed to `tutorials`
- Added logos

TODO:
- Enable github pages for the repo?
- Migrate solution booklet to markdown

qiaojunfeng committed:
Convert latex user guide to markdown

Using pandoc and mkdocs

qiaojunfeng committed:
Update

qiaojunfeng committed:
Switch to mathjax

To auto-number/crossref latex equations.

qiaojunfeng committed:
Add macro for `bm`

qiaojunfeng committed:
Use gfm flavor tables

qiaojunfeng committed:
Fix projections

qiaojunfeng committed:
Fix projections

qiaojunfeng committed:
Fix lib mode

qiaojunfeng committed:
Add overview.webp

qiaojunfeng committed:
Fix transport

jiang-yuha0 committed:
Fix markdown files for postw90 and appendices (#1)

* Update format for `Postw90` and `Appendices`
* Update equations and notes format

qiaojunfeng committed:
Fix files

jiang-yuha0 committed:
Fix markdown file (#2)

Fix mrkdown file wannier90/notes_interpolations.md

Commit:
Firt five tutorials for online version

Commit:
Firt five tutorials for online version

npaulish committed:
Fix markdown file for parameters chapter (#3)

* Fix markdown file for parameters chapter

* Fix a custom latex command

jiang-yuha0 committed:
Add Section postw90/parameters

jiang-yuha0 committed:
Add Section postw90/parameters

Commit:
17 tutorials for online version

Commit:
17 tutorials for online version

npaulish committed:
Fix tables in postw90/parameters.md file

Fix a reference in postw90params.md

npaulish committed:
Fix tables in postw90/parameters.md file

Fix a reference in postw90params.md

jiang-yuha0 committed:
Update some links

jiang-yuha0 committed:
Update some links

Commit:
24 tutorials for online version

Commit:
24 tutorials for online version

Commit:
Added tutorial.md

Commit:
Added tutorial.md

jiang-yuha0 committed:
Check the format of markdown files.

Fix the ref labels and links.
Unified the notes format.
Minor format fixes.

jiang-yuha0 committed:
Check the format of markdown files.

Fix the ref labels and links.
Unified the notes format.
Minor format fixes.

Commit:
29 tutorials for online version

Commit:
29 tutorials for online version

Commit:
Fixing rendering of equations in t18 e t27

Commit:
Fixing rendering of equations in t18 e t27

jiang-yuha0 committed:
Remove absolute number in cross-file equation ref

jiang-yuha0 committed:
Remove absolute number in cross-file equation ref

jiang-yuha0 committed:
Merge branch 'mkdocs' of github.com:qiaojunfeng/wannier90 into mkdocs

jiang-yuha0 committed:
Remove more absolute numbers

Remove absolute numbers in cross-file equation reference

Commit:
Up to tutorial 30 with all figures

Commit:
Up to tutorial 30 with all figures

npaulish committed:
Fix tutorials 32 and 33

npaulish committed:
Fix tutorials 32 and 33

qiaojunfeng committed:
Add custom css

qiaojunfeng committed:
Add custom css

npaulish committed:
Fix figures in Tutorials 32 and 33

npaulish committed:
Fix figures in Tutorials 32 and 33

jiang-yuha0 committed:
Fix the figure label

jiang-yuha0 committed:
Fix the figure label

npaulish committed:
Fix figure captions in Tutorials 32 and 33

npaulish committed:
Fix figure captions in Tutorials 32 and 33

npaulish committed:
Minor fixes in Tutorials 32 and 33

npaulish committed:
Minor fixes in Tutorials 32 and 33

Commit:
Up to tutorial 31 with all figures and bugfix

Commit:
Up to tutorial 31 with all figures and bugfix

Commit:
solving conflicts in mkdocs.yml

Commit:
solving conflicts in mkdocs.yml

Commit:
solving conflicts in mkdocs.yml

qiaojunfeng committed:
Merge tutorials

qiaojunfeng committed:
Merge tutorials

jiang-yuha0 committed:
Add link from user_guide to tutorial

Rename the example to tutorial in user_guide and add link from
user_guide to tutorial

jiang-yuha0 committed:
Fix link

Commit:
Fixing corss-references in the tutorials

Commit:
Fixing corss-references in the tutorials

jiang-yuha0 committed:
Fix caption in Tutorial 8

qiaojunfeng committed:
Merge branch 'mkdocs_merge' into mkdocs_merge

qiaojunfeng committed:
Merge pull request #6 from VVitale/mkdocs_merge

Mkdocs merge

qiaojunfeng committed:
Fix figcaption latex

Commit:
Changing all examples into tutorials, creating a markedown README and fixing some typos in the documentation

jiang-yuha0 committed:
Fix links

Commit:
Fixing typos in Tutorial 17 and Tutorial 8

Commit:
Added logos folder into the docs

npaulish committed:
Minor fixes in the docs

qiaojunfeng committed:
Set logo

qiaojunfeng committed:
Merge remote-tracking branch 'upstream/develop' into mkdocs

qiaojunfeng committed:
Cleanup mkdocs yaml

qiaojunfeng committed:
Fix precommit hook for mkdocs.yml

qiaojunfeng committed:
Add github workflow for mkdocs

qiaojunfeng committed:
Add CI for docs test

giovannipizzi reviewed (commented):
Thanks! A couple of small things
- let's remove all warnings and convert warnings to errors in the CI (with the `--strict` option of `mkdocs build`)
- let's add some content on the home page
- add a way to see the compiled docs of a PR

giovannipizzi committed:
Removing old latex and PDF

Since now tutorial has migrated to mkdocs/html,
we do not need to keep the old versions (that will
anyway get outdated if left there, and generate
confusion on which is the official documentation)

Commit:
fixing link to equation in tutorial 29

qiaojunfeng committed:
Fix md links

qiaojunfeng committed:
Enable strict mode of mkdocs

Commit:
Fixing paths to the correct tutorial folder on the github repo in the tutorials files

qiaojunfeng committed:
Update docs home

qiaojunfeng committed:
Change docs color

qiaojunfeng committed:
Use GITHUB_TOKEN

qiaojunfeng committed:
Merge remote-tracking branch 'upstream/develop' into mkdocs

giovannipizzi reviewed (approved):
Thanks! Great work @qiaojunfeng @VVitale @jiang-yuha0 @npaulish !
